src/pseudopeople/interface.py

```diff
@@ -79,7 +78,7 @@ def _generate_dataset(
         if not dataset_data:
             continue
 
-        noised_data = dataset_data.get_noised_data(configuration_tree, NOISE_TYPES)
+        noised_data = dataset_data._get_noised_data(configuration_tree, NOISE_TYPES)
```

Comment:
This shouldn't have been changed to have an underscore. It's a public method referenced outside the class!